Assets/Scripts/Core/Portal.cs

```diff
@@ -19,6 +19,7 @@ public class Portal : MonoBehaviour {
     Material firstRecursionMat;
     List<PortalTraveller> trackedTravellers;
     MeshFilter screenMeshFilter;
+    int portalSideOld;
```

Comment:
establish portalSideOld as an int that can be accessed script wide rather than only in the HandleTravellers function